Itm 790 variable definitions

kaitlyn-sharo opened the pull request:
Joint PR by Derek and me:

All variable definitions across the dashboard should match the tables, including differences between DRE and PH1. Downloads are now Excel instead of PDFs for quicker/easier editing (this was okayed by Jennifer).

Bug fix!
The formatTime function in resultsTable (the survey results table) was wrapping time, so if someone had spent an hour and 11 minutes, it would just show 11 minutes. I updated this so the function can optionally show hours, which it only will for Total Time. However, it no longer wraps minutes if hours are not shown. So for any crazy outliers on individual pages (one participant (I think 546) took over a dozen hours on a single page), minutes will continue increasing past 60 instead of wrapping around.

nextcen-dgemoets commented:
Now that we have different versions of the some of the tables for DRE vs. PH1, can we make the PH1 list a range of 0.2 - 0.8?

kaitlyn-sharo commented:
> Now that we have different versions of the some of the tables for DRE vs. PH1, can we make the PH1 ones list a range of 0.2 - 0.8?

We did, unless we missed one. Where are you seeing the problem?

nextcen-dgemoets commented:
> We did, unless we missed one. Where are you seeing the problem?

My mistake!  I thought the instance of 0.0 - 1.0 was in a "target" column, but it was in an "alignment score" column.

nextcen-dgemoets commented:
Maybe not worth the trouble, but it looks like RQ8 could have separate DRE/PH1 versions due to definitions of `Alignment score (Participant|high target)` and `Alignment score (Participant|low target)`, which list 1.0 and 0.0 in, e.g., "Calculated alignment score between the KDMA measurement of a participant and a high target (1.0 for ADEPT, qol-synth-HighExtreme, or vol-synth-HighExtreme)"

nextcen-dgemoets commented:
I feel like I've asked this before, but if RQ4 uses `Definitions_RQ1_RQ3_eval<n>.xlsx`, why isn't the file named `Definitions_RQ1_RQ4_eval<n>.xlsx`?

nextcen-dgemoets commented:
There are also a few typos in `mre_Definitions.xlsx`, but maybe we don't care about those at this point:
- `Misligned` -> `Misaligned`
- `ST_Attib_Grp` -> `ST_Attrib_Grp`
- `AD_Attib_Grp` -> `AD_Attrib_Grp`
- "low,based" -> "low, based"

kaitlyn-sharo commented:
> I feel like I've asked this before and got a perfectly good answer, but if RQ4 uses `Definitions_RQ1_RQ3_eval<n>.xlsx`, why isn't the file named `Definitions_RQ1_RQ4_eval<n>.xlsx`?

Those downloading it are typically only going to care about RQ 1 and RQ 3, as RQ 4 is part of the exploratory analysis

brianpippin commented:
> > I feel like I've asked this before and got a perfectly good answer, but if RQ4 uses `Definitions_RQ1_RQ3_eval<n>.xlsx`, why isn't the file named `Definitions_RQ1_RQ4_eval<n>.xlsx`?
> 
> Those downloading it are typically only going to care about RQ 1 and RQ 3, as RQ 4 is part of the exploratory analysis

I agree slightly with Darren that is a little weird that downloading RQ5 - 8 gives me different definitions files but downloading RQ4 gives me the RQ1 RQ3 file, so it could slighly be confusing.  I think if RQ4 is the same as the the RQ1 RQ3 file which it looks like it is, we should find a way to include RQ4 in the file name so it isn't too confusing for the users.

kaitlyn-sharo commented:
@nextcen-dgemoets Changed variable definitions and normal download for 134 to "134" instead of just 1 and 3; fixed the typos you mentioned in MRE (and removed some extra columns I found. I was confused on if there were more extras throughout, but I know Derek checked and I trust him, so it's probably good); and added the separate variables sections for RQ8 based on DRE/PH1

nextcen-dgemoets commented:
Everything looks good.  Still some minor stuff in the `mre_Definitions.xlsx` document:
- There are places where we refer to `AD_Attrip_Grp`/`SD_AttribGrp` instead of `AD_AttribGrp`/`SD_AttribGrp`.
- Also, there doesn't seem to be an `AD_AttribGrp`, but there is `AD_AttribGrp_Text` and `AD_AttribGrp_Sim`.  Maybe it's implied that it's referring collectively to each one?
  - e.g., "Calculate based on AD_AttribGrp and High/low designation of Medic A and B"
  - Same thing with or `SD_AttribGrp`.
- There's still a "low,based" that should be "low, based"
- For cell `O5`...is that deliberate, having `Debug.log` code?

kaitlyn-sharo commented:
> Everything looks good. Still some minor stuff in the `mre_Definitions.xlsx` document:
> 
> * There are places where we refer to `AD_Attrip_Grp`/`SD_AttribGrp` instead of `AD_AttribGrp`/`SD_AttribGrp`.
> * Also, there doesn't seem to be an `AD_AttribGrp`, but there is `AD_AttribGrp_Text` and `AD_AttribGrp_Sim`.  Maybe it's implied that it's referring collectively to each one?
>   
>   * e.g., "Calculate based on AD_AttribGrp and High/low designation of Medic A and B"
>   * Same thing with or `SD_AttribGrp`.
> * There's still a "low,based" that should be "low, based"
> * For cell `O5`...is that deliberate, having `Debug.log` code?

@dereknop

nextcen-dgemoets commented:
All of the changes look good.  Just noticed:
- Many cells in that excel refer to `AD_Align_Grp` or `ST_Align_Grp`, but I don't see that as a column header.  Was this removed, renamed or broken up into multiple columns at some point?

dereknop commented:
> All of the changes look good. Just noticed:
> 
> * Many cells in that excel refer to `AD_Align_Grp` or `ST_Align_Grp`, but I don't see that as a column header.  Was this removed, renamed or broken up into multiple columns at some point?

Old variable that got renamed. Replacing now

nextcen-dgemoets reviewed (approved):
Everything's been addressed.
